test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -2856,6 +2927,8 @@ TEST_P(CdsTest, WrongEdsConfig) {
   CheckRpcSendFailure();
   EXPECT_EQ(balancers_[0]->ads_service()->cds_response_state(),
             AdsServiceImpl::NACKED);
+  EXPECT_EQ("ConfigSource is not ADS.",
```

Comment:
This message should probably be changed to say "EDS ConfigSource is not ADS".  Otherwise, it's not clear which ConfigSource is being referred to.